consensus/e2e_tests/utils_test.go

```diff
@@ -3,6 +3,9 @@ package e2e_tests
 import (
 	"context"
 	"fmt"
+	"github.com/pokt-network/pocket/internal/testutil/p2p"
+	"github.com/pokt-network/pocket/internal/testutil/persistence"
```

Comment:
@h5law you add these import aliases manually or did an IDE do it? Just curious. I explicitly (unconventionally) named the package so that an import alias wouldn't be required but I didn't have a strong opinion on whether to use one anyways.

Reply:
I did I saw the errors as you were using `persistence_testutil` in the code and assumed this would be the import name

https://github.com/pokt-network/pocket/pull/753/files#diff-351d333685b4f6a9bb38d617a69a200daa5261f20ea31a62a7f1b4123dba9520R107